curated_tokenizers/_bbpe.pyx

```diff
@@ -66,6 +66,14 @@ cdef class ByteBPEProcessor:
             c_merges.push_back(pair[string, string](p1.encode('utf-8'), p2.encode('utf-8')))
         self._merges = make_shared[Merges](c_merges)
 
+    def __copy__(self):
+        return ByteBPEProcessor(vocab=self.vocab, merges=self.merges)
+
+    def __deepcopy__(self, memo):
+        result = ByteBPEProcessor(vocab=self.vocab, merges=self.merges)
```

Comment:
Maybe we should add a comment here why it is not necessary to deepcopy the vocab and merges.

Reply:
Added.